app.py

```diff
@@ -427,6 +426,18 @@ def _get_risk_level(debt_to_supply_ratio: float) -> str:
     streamlit.plotly_chart(figure_or_data=collateral_figure, use_container_width=True)
     streamlit.plotly_chart(figure_or_data=debt_figure, use_container_width=True)
 
+    collateral_protocol_color_map = {"zkLend": "#fd8d3c", 
```

Comment:
What if we change name for `zkLend` ?
for this reason, to have only one variable and use it everywhere we have PROTOCOL_NAMES:
feel free to create class and do it more maintainable:
what should you do, 
1. Create constant variable in app.py for each lending protocol
2. Substitute these constant variables in `PROTOCOL_NAMES`
3.Substitute `collateral_protocol_color_map`,  `collateral_protocol_color_map`, `collateral_protocol_color_map`

it should look like this one:
```
supply_protocol_color_map = {ZKLEND "#6baed6", 
                        NOSTRA_ALPHA: "#2171b5",
                        NOSTRA_MAINNET: "#08306b"}
```